src/MPAGSCipher/ProcessCommandLine.cpp

```diff
@@ -63,13 +61,13 @@ bool processCommandLine(const std::vector<std::string>& cmdLineArgs,
                 break;
             } else {
                 // Got the key, so assign the value and advance past it
-                cipherKey = cmdLineArgs[i + 1];
+                settings.cipherKey = cmdLineArgs[i + 1];
                 ++i;
             }
         } else if (cmdLineArgs[i] == "--encrypt") {
-            encrypt = true;
+            settings.encrypt = true;
```

Comment:
Should be using the `CipherMode` enumeration here:
```suggestion
            settings.cipherMode = CipherMode::Encrypt;
```